src/games/strategy/triplea/ui/NotesPanel.java

```diff
@@ -1,20 +1,15 @@
 package games.strategy.triplea.ui;
 
-import java.awt.Component;
 import java.awt.Rectangle;
 
 import javax.swing.BoxLayout;
-import javax.swing.JButton;
 import javax.swing.JEditorPane;
-import javax.swing.JLabel;
 import javax.swing.JPanel;
 import javax.swing.JScrollPane;
-import javax.swing.SwingUtilities;
 
 public class NotesPanel extends JPanel {
   private static final long serialVersionUID = 2746643868463714526L;
   protected final JEditorPane m_gameNotesPane;
-  final JButton m_refresh = new JButton("Refresh Notes");
```

Comment:
Wait - I was wrong, you didn't change anything there...
My bad...
But still this Button is not doing more than reinitializing the Pane...
The Refresh Button calls layoutNotes()
```
void layoutNotes() {
    if (m_gameNotesPane == null) {
      return;
    }
    removeAll();//This removes all components
    add(new JLabel(" "));//This is for style
    add(m_refresh);//This readds the button
    add(new JLabel(" "));//This is for style
    //This readds a JScrollPane with exact the same m_gameNotesPane as before, passed in the constructor...
    //no change to be expected...
    final JScrollPane scroll = new JScrollPane(m_gameNotesPane);
    scroll.scrollRectToVisible(new Rectangle(0, 0, 0, 0));//This forces the notes to start in top
    add(scroll);//this adds the JScrollPane to the JPanel
}
```